packages/common/src/store/account/sagas.ts

```diff
@@ -158,22 +151,27 @@ function* initializeMetricsForUser({
   }
 }
 
-export function* fetchAccountAsync() {
+export function* fetchAccountAsync({
+  shouldMarkAccountAsLoading
+}: {
+  shouldMarkAccountAsLoading: boolean
```

Comment:
Something about the pattern of passing a variable to tell a function what it should do internally feels a little icky, but I can't quite place it.
Almost wondering if we should just use two different actions, one for load and one for refresh, and they both use this function to do most of the work, but one sets loading state and the other doesn't?

I'm being super nitpicky. I just know there was a `isSignUp` variable being passed to this function forever and it wasn't used and didn't do anything :-D

Reply:
>Almost wondering if we should just use two different actions, one for load and one for refresh, and they both use this function to do most of the work, but one sets loading state and the other doesn't?

we actually need to call the saga vs. dispatch an action (because we need to wait for stuff). I would be down to split this into two sagas, one that responds to the action and the other is callable? how does that feel?

Reply:
Probably not worth it. I forgot that we call it directly in most places.